pkgs/development/interpreters/luajit/default.nix

```diff
@@ -3,32 +3,23 @@ rec {
 
   luajit = luajit_2_1;
 
-  luajit_2_0 = generic {
+  luajit_2_0 = (generic {
     version = "2.0.5";
     isStable = true;
     sha256 = "0yg9q4q6v028bgh85317ykc9whgxgysp76qzaqgq55y6jy11yjw7";
-  } // {
```

Comment:
It should be possible to not introduce `overrideAttrs` here:
```
luajit_2_0 = generic {
     version = "2.0.5";
     isStable = true;
     sha256 = "0yg9q4q6v028bgh85317ykc9whgxgysp76qzaqgq55y6jy11yjw7";
     meta = meta // {
      platforms = lib.filter (p: p != "aarch64-linux") meta.platforms;
 };

generic = { ..., meta ? meta }:
 stdenv.mkDerivation rec {
      inherit name version src meta;
....
;
```
`overrideAttrs` doesn't make this generative stuff cleaner IMO.

Reply:
I agree. @Infinisil can you adjust it as shown by @danbst?